.travis_target.sh

```diff
@@ -24,6 +24,7 @@ fi
 
 git clone https://github.com/puppetlabs/cpp-pcp-client
 cd cpp-pcp-client
+git checkout stable
```

Comment:
I think this should be a tag.  This is the travis setup for pxp-agent rather than an attempt to test-integrate on the fly.  Now we have cpp-pcp-client 1.0.0 we can just make this checkout of stable be a checkout of 1.0.0